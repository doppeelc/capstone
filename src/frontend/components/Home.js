import React from "react";


function Home() {
    return (
        <div>
            <h1>
                Factly
            </h1>
            <h3>
                Please Sign Up or Log In.
            </h3>
        </div>
    );
}

export default Home;