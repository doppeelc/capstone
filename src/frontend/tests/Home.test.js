import { render } from '@testing-library/react';
import Home from '../components/Home';

test('renders without crashing', () => {
    render(<Home />);
});

test('matches the snapshot', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
});






