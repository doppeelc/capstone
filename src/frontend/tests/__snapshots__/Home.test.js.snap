// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div>
    <h3>
      Please Sign Up or Log In.
    </h3>
  </div>
</DocumentFragment>
`;
